src/main/java/cc/hyperium/mods/guiscale/command/GuiscaleCommand.java

```diff
@@ -0,0 +1,33 @@
+package cc.hyperium.mods.guiscale.command;
+
+import cc.hyperium.Hyperium;
+import cc.hyperium.commands.BaseCommand;
+import cc.hyperium.commands.CommandException;
+import cc.hyperium.config.Settings;
+import cc.hyperium.utils.ChatColor;
+
+public class GuiscaleCommand implements BaseCommand {
+    @Override
+    public String getName() {
+        return "guiscale";
+    }
+
+    @Override
+    public String getUsage() {
+        return "/guiscale <integer>";
```

Comment:
```suggestion
        return "/guiscale <number>";
```
so people know what we talking about